src/main/java/org/broadinstitute/consent/http/db/DataAccessRequestDAO.java

```diff
@@ -35,69 +36,94 @@ public interface DataAccessRequestDAO extends Transactional<DataAccessRequestDAO
    */
   @UseRowReducer(DataAccessRequestReducer.class)
   @SqlQuery(
-      "SELECT dd.dataset_id, dar.id, dar.reference_id, dar.collection_id, dar.parent_id, dar.draft, dar.user_id, dar.create_date, dar.sort_date, dar.submission_date, dar.update_date, "
-          + "  (regexp_replace(dar.data #>> '{}', '\\\\u0000', '', 'g'))::jsonb AS data FROM data_access_request dar"
-          + "  LEFT JOIN dar_dataset dd on dd.reference_id = dar.reference_id "
-          + "  WHERE dar.draft != true "
-          + "  AND (LOWER(dar.data->>'status') != 'archived' OR dar.data->>'status' IS NULL)")
+      """
+          SELECT dd.dataset_id, dar.id, dar.reference_id, dar.collection_id, dar.parent_id, dar.user_id, dar.create_date, dar.sort_date, dar.submission_date, dar.update_date,
+            (regexp_replace(dar.data #>> '{}', '\\\\u0000', '', 'g'))::jsonb AS data FROM data_access_request dar
+          LEFT JOIN dar_dataset dd on dd.reference_id = dar.reference_id 
+          WHERE dar.submission_date is not null
+          AND (LOWER(dar.data->>'status') != 'archived' OR dar.data->>'status' IS NULL)""")
```

Comment:
Order of `OR` seems odd here, wouldn't you want to check for `IS NULL` before calling `LOWER()`? Or maybe SQL is smart enough to reverse these two expressions.

Reply:
Two things:
1) I've kept the syntax as-is because it's widely used and didn't want to creep scope on this.
2) I'm wondering if there are other optimizations we could be doing here to avoid the data mutation that happens in the select.  
[https://broadworkbench.atlassian.net/browse/DT-1563](https://broadworkbench.atlassian.net/browse/DT-1563)